src/frontend/sharedComponents/Map/UserLocation.tsx

```diff
@@ -0,0 +1,23 @@
+import MapboxGL from "@react-native-mapbox-gl/maps";
+import * as React from "react";
+import useSettingsValue from "../../hooks/useSettingsValue";
+
+interface UserLocationProps {
+  visible: boolean;
+  minDisplacement: number;
+}
+
+export const UserLocation = ({
+  visible,
+  minDisplacement,
+}: UserLocationProps) => {
+  const directionalArrow = useSettingsValue("directionalArrow") as boolean;
```

Comment:
Can avoid the type assertion here by updating `useSettingsValue` here: https://github.com/digidem/mapeo-mobile/pull/914/files#r790968391